web/apps/apps.go

```diff
@@ -273,7 +271,7 @@ func logsHandler(appType consts.AppType) echo.HandlerFunc {
 			WithField("slug", slug).
 			WithField("job_id", c.QueryParam("job_id"))
 
-		if clientSide == true {
+		if clientSide {
```

Comment:
We could simply do `if man.ClientSide() {` now.

Reply:
No, we can't, `man` is not in the scope here

Reply:
Ah sorry, I missed the folded lines.
The assignment can be simplified though with `clientSide = man.ClientSide()`